apps/taquito-test-dapp/README.md

```diff
@@ -4,13 +4,18 @@
 
 A minimal end-to-end testing setup for developing Tezos DApps with Taquito and Beacon to manage signing and wallet operations.
 ## Getting Started
+#### Initial setup:
 1. Clone the Taquito repository: `git clone git@github.com:ecadlabs/taquito.git`
-2. Change your current working directory to the newly cloned repository directory, then change to `apps/taquito-test-dapp`.
-3. Install all dependencies:
-    `npm clean-install`
-4. Start development server:
-    `npm run dev`
-5. Open http://localhost:3030 in your browser to see a sample application.
-6. Start production server:
-    `npm run build && npm run preview`
-7. Open http://localhost:4173 in your browser to see a preview application.
+1. Change your current working directory to the newly cloned repository.
+1. Install dependencies: `npm i`
+1. Build Taquito: `npm run build`
+1. Change your current working directory to `apps/taquito-test-dapp`.
+1. Start development or production server as shown below.
+
+#### Start development server:
+1. `npm run dev`
+1. Open http://localhost:3030 in your browser to see a sample application.
+
+#### Start production server:
+1. `npm run build && npm run preview`
+1. Open http://localhost:4173 in your browser to see a preview application.
```

Comment:
And thank you for the above: it's a nice improvement!

Reply:
Thanks for the quick review!